app/src/componentLibrary/tooltip/tooltip.jsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ReactDOM from 'react-dom';
+import { Popper } from 'react-popper';
+import styles from './tooltip.scss';
+
+const cx = classNames.bind(styles);
+
+export const Tooltip = (props) => {
+  const {
+    children,
+    dynamicWidth,
+    width,
+    topOffset,
+    leftOffset,
+    side,
+    modifiers,
+    noArrow,
+    dataAutomationId,
+  } = props;
+  const styleWidth = dynamicWidth ? null : { width };
+  const clientWidth = document.documentElement.clientWidth;
+  const maxWidth = !styleWidth ? clientWidth - 100 : styleWidth;
+  const tooltipRoot = document.getElementById('tooltip-root');
+  return ReactDOM.createPortal(
+    <Popper placement={side} modifiers={modifiers} eventsEnabled={false}>
+      {({ placement, ref, style, arrowProps }) => (
+        <div
+          className={cx('tooltip')}
+          ref={ref}
+          style={{
+            ...style,
+            ...styleWidth,
+            top: style.top + topOffset,
+            left: style.left + leftOffset,
+          }}
+          data-placement={placement}
+          data-automation-id={dataAutomationId}
+        >
+          <div
+            className={cx('tooltip-content')}
+            style={{
+              maxWidth: `${maxWidth}px`,
+            }}
+          >
+            {children}
+            {!noArrow && (
+              <div
+                className={cx('tooltip-arrow')}
+                data-placement={placement}
+                ref={arrowProps.ref}
+                style={arrowProps.style}
+              />
+            )}
+          </div>
+        </div>
+      )}
+    </Popper>,
```

Comment:
Why are you decided to move it to another file?
AFAIK the `Popper` must be used inside of `Manager`, so it is better to use them together in one file.
The one file will be bigger, but in this case it will be easier to see the connection between elements.